pages/activity/[id]/index.tsx

```diff
@@ -40,6 +40,22 @@ const { t } = i18n;
 
 const MainForumName = '主论坛';
 
+const shareInvitation = () => {
+  if (!navigator.canShare) {
+    //不支持
+    console.log('no support share');
+  } else if (
+    navigator.canShare({
```

Comment:
```suggestion
  if (
    navigator.canShare?.({
```
https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Operators/Optional_chaining#%E5%8F%AF%E9%80%89%E9%93%BE%E4%B8%8E%E5%87%BD%E6%95%B0%E8%B0%83%E7%94%A8

Reply:
```ts
const shareInvitation = async () => {
  try {
    await navigator.share?.({
      title: 'web.dev',
      text: 'Check out web.dev.',
      url: 'https://web.dev/',
    });
    console.log('share success');
  } catch (error) {
    console.log('share error', error);
  }
};
```
我改成这样